### PR TITLE
maclaunch 2.3.1 (new formula)

### PR DESCRIPTION
**Pull request by hazcod**
Copied from https://raw.githubusercontent.com/hazcod/homebrew-hazcod/master/Formula/maclaunch.rb

Repo: https://github.com/hazcod/maclaunch
Lists and controls all your macOS startup items and their startup policy.

- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/maclaunch.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://github.com/hazcod/maclaunch/archive/2.3.1.tar.gz"
+  sha256 "abba1f7cffd7f694b23745f6ccc137b17b6c9ea38fe2fbb55a8bd9646f6ae1a1"
+  license "MIT"
+
+  def install
+    bin.install "maclaunch.sh" => "maclaunch"
+  end
+
+  test do
+    system "#{bin}/maclaunch" list
```
Comment (SMillerDev) [inline Formula/maclaunch.rb:13]:
```suggestion
    system bin/"maclaunch", "list"
```

**Comment by SMillerDev** [comment]
I don't think the test is working quite correct. It also runs on Linux.

**Comment by hazcod** [comment]
> I don't think the test is working quite correct. It also runs on Linux.

Can you elaborate? It's a macOS tool, how do I skip linux for tests?

**Comment by SMillerDev** [comment]
Seeing how the tests are passing on linux I don't think the tests work. Either way, you can add `depends_on :macos` to make sure it's only tested on macOS

**Comment by hazcod** [comment]
@SMillerDev I think that should all be OK now.

**Review by chenrui333 — APPROVED** [review]

**Comment by hazcod** [comment]
LGTM! 🙌🏼

**Comment by chenrui333** [comment]
@hazcod, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**